test/functional/test_framework/test_node.py

```diff
@@ -434,7 +434,7 @@ def batch(self, requests):
     def send_cli(self, command=None, *args, **kwargs):
         """Run bitcoin-cli command. Deserializes returned string as python object."""
         pos_args = [str(arg).lower() if type(arg) is bool else str(arg) for arg in args]
-        named_args = [str(key) + "=" + str(value) for (key, value) in kwargs.items()]
+        named_args = [str(key) + "=" + (str(value).lower() if type(value) is bool else str(value)) for (key, value) in kwargs.items()]
```

Comment:
Same comment here as https://github.com/bitcoin/bitcoin/pull/15235#discussion_r250547555

Reply:
Since this issue affects multiple PRs, I've split it into a separate PR: #15301